examples/classifcation/text_classification.exs

```diff
@@ -0,0 +1,273 @@
+# Based on https://valueml.com/named-entity-recognition-using-lstm-in-keras/
+
+# References
+# http://alexminnaar.com/2019/08/22/ner-rnns-tensorflow.html
+
+# Test Data: https://github.com/bhuvanakundumani/NER_tensorflow2.2.0/blob/master/data/test.txt
+
+# Train Data: https://github.com/bhuvanakundumani/NER_tensorflow2.2.0/blob/master/data/train.txt
+
+# Validation Data: https://github.com/bhuvanakundumani/NER_tensorflow2.2.0/blob/master/data/valid.txt
+
+Mix.install([
+  {:axon, github: "elixir-nx/axon"},
+  {:nx, "~> 0.2.1"},
+  {:exla, "~> 0.2.2"}
+])
+
+EXLA.set_as_nx_default([:tpu, :cuda, :rocm, :host])
+
+defmodule TextClassification do
+  require Axon
+  # From Example http://alexminnaar.com/2019/08/22/ner-rnns-tensorflow.html
+  EXLA.set_as_nx_default([:tpu, :cuda, :rocl, :host]) # My :host option doesnt work, jit_apply is undefined
+  require Logger
+  @sequence_length 75
+  @batch_size 128
+  @lstm 128
+  @embed_dimension 256
+  @def_split " "
+  @start "-DOCSTART- -X- -X- O\n"
+  @ending "-------------------------------------------------------------"
+  @incorrect [".", "?", "%", ",", ":", ";", "-docstart-"]
+
+  # seqlen
+  # word_count
+
+  def train(model, data) do
+    model
+    |> Axon.Loop.trainer(&kl_divergence/2, Axon.Optimizers.adam(0.001))
+    |> Axon.Loop.metric(:accuracy, "Accuracy")
+    |> Axon.Loop.run(data, %{}, epochs: 3, iterations: 5)
```

Comment:
Only 5 iterations?

Reply:
Oversight from early testing, I've been running at 100,300,500,600,800 iterations at current, way back at the start i wanted to see what was actually happening with the model loading & saving etc, was just easy for testing. This is something I need to also cover a bit more in the live book as it's an important key factor to understand I believe (I'm completely self taught other than what I've learnt from yourself, recent mentoring in the space). I'll need to do some more research into how elaborate on the hyper-parameters as a whole